### PR TITLE
[1174] load mcb sub-commands automatically

### PR DESCRIPTION
**Pull request by misaka**
### Context

We will be adding commands to `mcb` for importing, this change makes is simpler and less error-prone to do this.

### Changes proposed in this pull request

Automatically load all the commands and sub-commands in `lib/mcb/commands` instead of having to list them all out in `bin/mcb` manually.

### Guidance to review

Hey look! A spec for mcb! And a module too!!

### CODE REVIEW

File: spec/bin/mcb_spec.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
```
Comment (defong) [inline spec/bin/mcb_spec.rb]: Is there a point with `rails_helper`?

Just thinking its a console app no longer a rails app

**Comment by defong** [comment]
[Provider preferences - mcb importer](https://trello.com/c/6Iz0uWm8/1174-provider-preferences-mcb-importer)

File: spec/bin/mcb_spec.rb (hunk 1 above)
Comment (misaka) [reply spec/bin/mcb_spec.rb]: Good spot.

Some commands _will_ load the Rails environment, though, but I guess we can worry about those later.

File: spec/lib/mcb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
+require 'spec_helper'
+require "#{Rails.root}/lib/mcb"
+
+describe 'mcb command' do
+  describe '#load_commands' do
+    include FakeFS::SpecHelpers
```
Comment (defong) [inline spec/lib/mcb_spec.rb]: is there something we can about 
` include FakeFS::SpecHelpers`

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -5,6 +5,7 @@
 require "super_diff/rspec"
 require 'support/matchers/have_courses'
 require 'support/matchers/have_providers'
+require 'fakefs/spec_helpers'
```
Comment (defong) [inline spec/spec_helper.rb]: That was the thing i was thinking about for 

` include FakeFS::SpecHelpers`

**Pull request merged**